src/MonitoringFunctions/Models/HttpRequestLogEntry.cs

```diff
@@ -8,7 +9,7 @@ namespace MonitoringFunctions.Models
     /// <summary>
     /// Represents an http request event made from a function to be inserted into Kusto.
     /// </summary>
-    internal class HttpRequestLogEntry
+    internal class HttpRequestLogEntry : IKustoTableRow
```

Comment:
if these will be short-lived instances, consider a struct (if then also implement IEquatable

Reply:
please resolve at your prerogative